wait/host_port_test.go

```diff
@@ -508,3 +508,58 @@ func TestHostPortStrategySucceedsGivenShellIsNotInstalled(t *testing.T) {
 	err = wg.WaitUntilReady(context.Background(), target)
 	require.NoError(t, err)
 }
+
+func TestHostPortStrategySucceedsGivenShellIsNotFound(t *testing.T) {
```

Comment:
cool, thanks for this! Do you think it's possible to add a test for the internal function so that we verify the new error is raised?

Reply:
done.